cmake/projects/soil/hunter.cmake

```diff
@@ -0,0 +1,19 @@
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    soil
+    VERSION
+    1.0.4
+    URL
+    "https://github.com/frequem/Simple-OpenGL-Image-Library/archive/v1.0.4.tar.gz"
```

Comment:
Do you want me to create https://github.com/hunter-packages/Simple-OpenGL-Image-Library instead? See https://docs.hunter.sh/en/latest/creating-new/patch.html for details.

Reply:
Not really necessary since it's not explicitly a hunter version and can also be used in standalone cmake.